setup.cfg

```diff
@@ -16,6 +16,7 @@ max-line-length = 88
 max-doc-length = 200
 per-file-ignores =
     ./keras_cv/__init__.py:E402, F401
+    ./keras_cv/examples/training/*:E402
```

Comment:
You'll need to do: ./examples/*

./keras_cv/examples doesn't exist

Reply:
Lets do all examples, you might need to do examples/**/* to achieve this.

Reply:
Hmm. I thought I had tested this but it turned out I hadn't correctly caused the error in the file I was testing with. I've updated this to ./examples/**/* and it now seems to be working correctly.